cmd/minikube/cmd/start.go

```diff
@@ -1210,3 +1202,10 @@ func getKubernetesVersion(old *config.ClusterConfig) string {
 
 	return version.VersionPrefix + nvs.String()
 }
+
+func skipErrorIfForceEnabled(code int, message string, v out.V) {
```

Comment:
rename to something more readable, maybe

"exitIfPossible" 
"exitIfNotForced"
"maybeExit"

Reply:
done